src/main/java/org/dormitory/autobotsoccub/command/querycommand/RegisterCommand.java

```diff
@@ -5,26 +5,42 @@
 import org.dormitory.autobotsoccub.command.keyboard.KeyboardFactory;
 import org.dormitory.autobotsoccub.command.keyboard.Button;
 import org.dormitory.autobotsoccub.command.result.CommandResult;
+import org.dormitory.autobotsoccub.user.UserPool;
 import org.telegram.telegrambots.api.objects.Update;
+import org.telegram.telegrambots.api.objects.User;
 
-import static org.dormitory.autobotsoccub.command.keyboard.Button.UNREGISTER;
-
+import static org.dormitory.autobotsoccub.command.keyboard.Button.*;
 
 @AllArgsConstructor
 public class RegisterCommand implements InlineQueryCommand {
 
     private KeyboardFactory keyboardFactory;
 
+    private UserPool userPool;
+
     @Override
     public Button getCommandButton() {
         return Button.REGISTER;
     }
 
     @Override
     public CommandResult execute(Update update) {
+        User currentUser = update.getCallbackQuery().getFrom();
+        if(userPool.containsUser(currentUser.getId())) {
+            return CommandResult.builder()
+                    .replyMessage(currentUser.getFirstName() + " you are already registered!")
```

Comment:
[future] this message is gonna spam the whole chat, empty reply might be better.

Reply:
With an empty response will be thrown telegtamApiExeption(also with empty message or space), so I propose to leave it as it is